packages/scrollable-video/src/components/video.js

```diff
@@ -214,7 +214,7 @@ const ForwardRefVideo = React.forwardRef(
     return (
       <Video
         muted
-        playsinline
```

Comment:
Is `playsInline` or `playsinline` correct?

In this ref (https://support.google.com/displayvideo/answer/9338186?hl=zh-Hant), it uses `playsinline`.

But I found `playsInline` in [caniuse results](https://caniuse.com/mdn-api_htmlvideoelement_playsinline)

Reply:
Since[ DOM attributes should be camelCased in React](https://github.com/facebook/react/issues/1649#issuecomment-45280929), `playsInline` seems to be the correct one.

I believe `playsInline` is mostly for iPhone, <video playsinline> elements will be allowed to play inline, and will not automatically enter fullscreen mode when playback begins. 

In caniuse results, `playsinline` is supported on iOS10 and latter, it meets our requirement.

Reply:
Oops, I forgot that React uses camelCase in the attributes.
Thanks for reminding.